paparazzi-gradle-plugin/src/test/java/app/cash/paparazzi/gradle/PaparazziPluginTest.kt

```diff
@@ -777,8 +777,8 @@ class PaparazziPluginTest {
       .runFixture(fixtureRoot) { build() }
 
     val snapshotsDir = File(fixtureRoot, "build/reports/paparazzi/debug/images")
-    val snapshots = snapshotsDir.listFiles()
-    assertThat(snapshots!!).hasLength(3)
+    val snapshots = snapshotsDir.listFilesSorted()
```

Comment:
nit: this adds an unnecessary array copy now.  I don't see the value in adding this everywhere, since it still needs to be a thing to remember when adding a new test.

Reply:
sure happy to only update the ones that have multiple images to compare instead.